### DISCUSSION TITLE
SQLite backend proposal

### DISCUSSION THREAD

**Discussion by wagga40** [SigmaHQ/pySigma#144] [Show and tell]
Hi, 

I've built a SQLite backend for pySigma. For now, it is hosted [here](https://github.com/wagga40/pySigma-backend-sqlite) but as proposed [here](https://github.com/SigmaHQ/cookiecutter-pySigma-backend) I would like to host the backend within the [SigmaHQ organization](https://github.com/SigmaHQ).

The rules generated by this new backend have been tested quite successfully with [Zircolite](https://github.com/wagga40/Zircolite) and compared to previous rules generated with the legacy converter.

**Comment by thomaspatzke** [comment]
Cool! Thanks for your great contribution! The backend was already demanded by various people and I've pushed already some notifications out to them 😉

I would propose that:

1. I fork your backend into the SigmaHQ oganization.
2. Give you full access to it.
3. You delete your original repo or keep it, as you want.

Should we go this way?

**Reply by wagga40** [reply]
Perfect 👍

thank you !

**Reply by thomaspatzke** [reply]
Done! It took me a while, but now it's in SigmaHQ and you have full admin permission on it!
